apps/backoffice-v2/src/pages/Entity/components/EditableDetails/EditableDetails.tsx

```diff
@@ -91,6 +91,13 @@ export const EditableDetails: FunctionComponent<IEditableDetails> = ({
         const properties = Object.keys(document?.propertiesSchema?.properties ?? {}).reduce(
           (acc, curr) => {
             if (!formData?.[curr]) return acc;
+
+            if (document?.propertiesSchema?.properties?.[curr]?.format === 'date-time') {
```

Comment:
Can you leave a comment explaining this fix?

Reply:
Yes, sorry: HTML5 `datetime-local` input holds a date time value. It should return `YYYY-MM-DDTHH:MM:SS`, but there is an edge-case when the seconds are `00`, it returns `YYYY-MM-DDTHH:MM` instead.
So here I fixed it, ensuring that when there are no seconds, I add `:00` to the string: `YYYY-MM-DDTHH:MM:00`